aretomo/protocols/protocol_aretomo.py

```diff
@@ -620,23 +599,21 @@ def createOutputTs(self, tsId: str, tsFn: str):
         # unless making a tomo from pre-aligned TS
         if not (self.makeTomo and self.skipAlign):
             outputSetOfTiltSeries = self.getOutputSetOfTiltSeries(OUT_TS)
-            newTs = ts.clone()
```

Comment:
It's cleaner and I think less expensive computationally to instantiate an empty object and fill it with a copyInfo call rather than clone it and call copyInfo